7_react/react1/week2/hw-react-week2/src/components/Container.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+
+import TaskList from "./TaskList";
+import FormTodo from "./FormTodo";
+
+const Container = () => {
```

Comment:
Try to give your component a more descriptive name if they are used for anything specific. `Container` doesn't say much about what it is used for. i.e. `TaskListContainer ` could be a more descriptive name.

Reply:
It makes sense, thanks for the suggestion. One question: it's straightforward now because I don't have many references to the name, and I can replace the names one by one. But is there any way for Visual Studio Code to modify all the references when I change the name of a component? thanks :)